### ISSUE TITLE
Command "make" errors out on windows

### ISSUE THREAD

**Issue by jreid2454** [MathCancer/PhysiCell#228]
I was following along with the tutorial here: https://www.youtube.com/watch?v=hIP4JUrViRA&t=653s

And I followed all of the steps, but when I got to the point where you just run "make", right after running "make biorobots-sample", things error out. Here is a snippet of my output from the terminal

(base) D:\physicell_v2\PhysiCell>make
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_vector.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_mesh.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_microenvironment.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_solvers.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_matlab.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_utilities.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_basic_agent.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_MultiCellDS.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/BioFVM_agent_container.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./BioFVM/pugixml.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./core/PhysiCell_phenotype.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./core/PhysiCell_cell_container.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./core/PhysiCell_standard_models.cpp
g++ -march=native  -O3 -fomit-frame-pointer -mfpmath=both -fopenmp -m64 -std=c++11  -c ./core/PhysiCell_cell.cpp
./core/PhysiCell_cell.cpp: In destructor 'virtual PhysiCell::Cell::~Cell()':
./core/PhysiCell_cell.cpp:441:56: error: no match for 'operator<<' (operand types are 'std::basic_ostream<char>' and 'std::vector<double>')
   std::cout << "I am of type " << this->type << " at " << this->position << std::endl;
                                                        ^
In file included from C:/ProgramData/Miniconda3/Library/mingw-w64/include/c++/5.3.0/iostream:39:0,
                 from ./core/./PhysiCell_custom.h:71,
                 from ./core/PhysiCell_cell.h:71,
                 from ./core/PhysiCell_cell.cpp:68:
C:/ProgramData/Miniconda3/Library/mingw-w64/include/c++/5.3.0/ostream:108:7: note: candidate: std::basic_ostream<_CharT, _Traits>::__ostream_type& std::basic_ostream<_CharT, _Traits>::operator<<(std::basic_ostream<_CharT, _Traits>::__ostream_type& (*)(std::basic_ostream<_CharT, _Traits>::__ostream_type&)) [with _CharT = char; _Traits = std::char_traits<char>; std::basic_ostream<_CharT, _Traits>::__ostream_type = std::basic_ostream<char>]
       operator<<(__ostream_type& (*__pf)(__ostream_type&))
       ^
C:/ProgramData/Miniconda3/Library/mingw-w64/include/c++/5.3.0/ostream:108:7: note:   no known conversion for argument 1 from 'std::vector<double>' to 'std::basic_ostream<char>::__ostream_type& (*)(std::basic_ostream<char>::__ostream_type&) {aka std::basic_ostream<char>& (*)(std::basic_ostream<char>&)}'
C:/ProgramData/Miniconda3/Library/mingw-w64/include/c++/5.3.0/ostream:117:7: note: candidate: std::basic_ostream<_CharT, _Traits>::__ostream_type& std::basic_ostream<_CharT, _Traits>::operator<<(std::basic_ostream<_CharT, _Traits>::__ios_type& (*)(std::basic_ostream<_CharT, _Traits>::__ios_type&)) [with _CharT = char; _Traits = std::char_traits<char>; std::basic_ostream<_CharT, _Traits>::__ostream_type = std::basic_ostream<char>; std::basic_ostream<_CharT, _Traits>::__ios_type = std::basic_ios<char>]
       operator<<(__ios_type& (*__pf)(__ios_type&))
       ^

It repeats this over and over again, in many different places. It seems like their is a missing library or something? Please help if you have any ideas what is wrong.

Also, I am running this on windows 10, and I was using the Anaconda terminal as shown in the video.

**Comment by rheiland** [comment]
The latest setup slides and video are at:
https://github.com/physicell-training/ws2023/blob/main/agenda.md#windows-setup

Can you provide the version of g++ you're using:  "g++ --version"

and the version of PhysiCell (the text in VERSION.txt in the root directory)

**Comment by jreid2454** [comment]
I am using g++ version 5.3.0, and PhysiCell version 1.13.1

**Comment by jreid2454** [comment]
Also, I uninstalled anaconda and msys2, removed the paths I had made for a fresh start. Then I followed along the tutorial slides you had linked to, and re did it all exactly as shown in the slides, and I am getting the same error. g++ and PhysiCell versions are the same.

**Comment by rheiland** [comment]
Something is wrong. Your g++ should be much newer. Can you copy/paste
exactly what you see? This is what I get:

PS C:\Users\randy\dev\PhysiCell-1.13.1> g++ --version
g++.exe (Rev6, Built by MSYS2 project) 13.1.0
Copyright (C) 2023 Free Software Foundation, Inc.

**Comment by jreid2454** [comment]
That was the issue! On my Anaconda terminal it says this

(base) C:\Users\Joshua Reid>g++ --version
g++ (Rev5, Built by MSYS2 project) 5.3.0
Copyright (C) 2015 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Which I suppose I should update, but then I checked my version on my normal windows command and I got this

C:\Users\Joshua Reid>g++ --version
g++ (Rev3, Built by MSYS2 project) 13.2.0
Copyright (C) 2023 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

And when I ran "make" it successfully created the executable file, which ran fine. 

Thanks so much for the help!
